zetaclient/evm/evm_client.go

```diff
@@ -1161,13 +1127,6 @@ func (ob *ChainClient) observeERC20Deposited(startBlock, toBlock uint64) uint64
 // returns the last block successfully scanned
 func (ob *ChainClient) observeTssRecvd(startBlock, toBlock uint64, flags observertypes.CrosschainFlags) uint64 {
```

Comment:
Can we do some renaming here?
```
ObserverTSSReceive
```

Reply:
https://github.com/zeta-chain/node/pull/1836/commits/be3e8f9862d885050fb7ac19a3845e62bc8b27cd